package.json

```diff
@@ -13,6 +13,7 @@
     "@viem/anvil": "^0.0.6",
     "@vitest/coverage-v8": "^0.34.2",
     "@wagmi/cli": "^1.3.0",
+    "abitype": "^0.9.8",
```

Comment:
do we need this or do we get it already from viem?

Reply:
I think can get it from viem